src/pages/AddMembersPage/NewMemberList.test.tsx

```diff
@@ -0,0 +1,180 @@
+import { act, fireEvent, render, screen } from '@testing-library/react';
+import { client } from 'lib/gql/client';
+
+import { TestWrapper } from 'utils/testing';
+
+import { Group } from './AddMembersPage';
+import NewMemberList from './NewMemberList';
+
+const save = async () => {
+  return [];
+};
+
+const group: Group = {
+  id: 1,
```

Comment:
I'd use a number rather than one like 89 or something that is clearer this is the org _id and not just a default value of 1.